puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/foreman.groovy

```diff
@@ -1,9 +1,7 @@
 def databaseFile(id, database = 'postgresql') {
```

Comment:
I think this function is still used by multiple versions so I'd prefer to keep this in until we don't test any mysql version anymore.

Reply:
this is used by `release/source/foreman.groovy`, where i removed the mysql stage, and `release/source/katello.groovy` and `test/testKatello.groovy` which don't support mysql already so i think it's pretty safe